aliases/available/dnf.aliases.bash

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+#zuck007
+cite 'about-alias'
+about-alias 'dnf aliases for fedora 22+ distros'
+
+#set-dnf-aliases
+function _set_dnf_aliases() 
```

Comment:
Ah, got it. We don't have a standard for this, but I would prefer to use it without the wrapper function.